src/TestInfrastructure/app.config

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file seems to be added due to the change to .NET4.6. Can anyone tell me if it is required? `Microsoft.IdentityModel.Clients.ActiveDirectory` and `Newtonsoft.Json` seems to be added to `Integration.Vsix.UnitTests` too, and it was there before. 

By the way we are using version 7.0.1 of Newtonsoft.Json, so this redirect seems wrong.

cc @duncanpMS